tests/linearizability/model.go

```diff
@@ -47,8 +47,8 @@ type EtcdResponse struct {
 }
 
 type EtcdState struct {
-	Key            string
-	PossibleValues []ValueRevision
+	Revision  int64
```

Comment:
Based on the implementation in this PR, it seems no need to keep `Revision` in `EtcdState`?